ansible.cfg

```diff
@@ -9,3 +9,6 @@ connection_plugins = plugins/connection
 callback_plugins = plugins/callbacks
 transport = monkeypatched_ssh
```

Comment:
Pipelining reduces the number of connections Ansible makes. In the default setup, ansible will ssh once to create a temporary directory, ssh again to write out a composed module file (module code plus arguments plus boilerplate code from ansible core). A third ssh is done to execute the module and clean up the temporary directory.

With pipelining, ansible opens one ssh directly to the python interpreter, and then "pipes" the composed module code across that connection for execution. Instead of 3 connections per task per host, it's one. 

This saved us HUGE amounts of time at Rackspace, but we couldn't use ControlPersist (too many hosts, sockets ran ansible host out of memory), and we had a... large number of hosts. Still, even with our typical 3 host setup, this could be a time saver.